content/composer/overview.md

```diff
@@ -9,39 +9,78 @@ weight: 200
 
 Cloudify Composer is an editor for creating Blueprint YAML files dynamically, using a drag and drop interface. 
 
-Cloudify Composer enables you to model topology for complex applications, and to add relevant lifecycle operations implementation via external plugins and scripts. 
+Cloudify Composer enables you to model topology for complex applications, and to add relevant lifecycle operations implementation by importing plugins and scripts, or packaging them with the blueprint itself. 
 
 Among its drag-and-drop components are platform and network items such as `compute` node, `database`, `web server`, and so on. You can also add your own custom node type components, custom plugins and interfaces. 
 
-The generated output from Cloudify Composer is a downloadable TGZ archive containing a blueprint.yaml *file* that provides a TOSCA-based description for the application topology and its lifecycle management. In addition to YAML artifacts, the blueprint archive includes a blueprint *package* that contains multiple resources such as configuration and installation scripts (or Puppet Manifests, or Chef Recipes, etc..), code, and basically any other resource you require for running your application.
+The generated output from Cloudify Composer is a downloadable TGZ or ZIP archive containing a blueprint.yaml *file* that provides a TOSCA-based description for the application topology and its lifecycle management. In addition to YAML artifacts, the blueprint archive includes a blueprint *package* that contains multiple resources such as configuration and installation scripts, code, and basically any other resource you require for running your application.
 
 ## Logging In
-The first time that you use Cloudify Composer, you must supply a username and a password. Use the following defaults:<br />
-**Username:** `composer` <br />
-**Password:** `composer`
 
-When you have logged in to Cloudify Composer, the Topology page for the selected blueprint is displayed.
+To use the Cloudify Composer component, you log into Cloudify Manager. If you are already in Cloudify Manager, click the ** Cloudify Composer* button in the **Local blueprints** screen.
```

Comment:
row 20 is incorrect: you don't login to the manager to use the Composer. you CAN - if you are logged in - use the link button, but you will just be directed to the Composer's separate login screen. The phrasing in the google doc is more accurate, and also users need to know they can access the composer using the specified ports. The comment in the google doc also emphasizes that the user management in Composer is the same on as Cloudify's, which tells the users they don't need to create new users, and that the Cloudify permissions will apply in Composer as well.